src/components/KYCWall/BaseKYCWall.tsx

```diff
@@ -279,6 +279,7 @@ export default withOnyx<BaseKYCWallProps, BaseKYCWallOnyxProps>({
     bankAccountList: {
         key: ONYXKEYS.BANK_ACCOUNT_LIST,
     },
+    // @ts-expect-error: ONYXKEYS.REIMBURSEMENT_ACCOUNT is conflicting with ONYXKEYS.FORMS.REIMBURSEMENT_ACCOUNT_FORM
```

Comment:
This is a big issue we have with these two keys, both have the same value but different types, which mean we keep two different types of data under one key. Ultimately it would be ideal to change one of these key to something else. I asked @barttom on slack if it's possible to refactor it in [Bank Refactor PR](https://github.com/Expensify/App/pull/34498)

Reply:
Should be resolved after this: https://expensify.slack.com/archives/C01GTK53T8Q/p1707150558368119